app/text_buffer.py

```diff
@@ -1289,6 +1289,17 @@ def doSelectionMode(self, mode):
           mode - self.selectionMode)))
       self.redo()
 
+  def selectCurrentLine(self):
```

Comment:
Consider naming this cursorSelectLine. This will be consistent with a collection of cursorSelect* functions.

(The naming will make more sense once cursorRow/Col is correctly separated from penRow/Col).